src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -533,6 +533,9 @@ void PeakDetectorCLI::loadCompoundsFile() {
 		int loadCount = DB.loadCompoundCSVFile(mavenParameters->ligandDbFilename);
 		mavenParameters->compounds = DB.compoundsDB;
 		cout << "Total Compounds Loaded : " << loadCount << endl;
+	} else if (!mavenParameters->processAllSlices) {
+		cerr << "\nPlease provide a compound database file to proceed with targeted analysis." << endl;
```

Comment:
@saifulbkhan Also add that they can use the -h option to get help. User might not remember all parameters and their usage.